plugins/modules/ovirt_nic.py

```diff
@@ -122,6 +135,19 @@
     id: 00000000-0000-0000-0000-000000000000
     name: "new_nic_name"
     vm: myvm
+
+# Add NIC network filter parameters
+- @NAMESPACE@.@NAME@.ovirt_nic:
+    state: present
+    name: mynic
+    vm: myvm
+    network_filter_parameters:
+      - name: GATEWAY_MAC
```

Comment:
What is the 'gateway_mac' filter? I don't think several gateways can be applied to the same vnic without a rules table.

Reply:
Configure MAC filtering on a **vNIC profile** with the **clean-traffic-gateway** filter.
The advantage is to place several VMs in the same subnet but to prevent them from talking to each other. The only authorized flow is the gateway(s).

[The GATEWAY MAC setting is documented here](https://libvirt.org/formatnwfilter.html#concepts).

First you need to have this:

```yaml
- name: Editing vNICs network_filter
  ovirt.ovirt.ovirt_vnic_profile:
    name: myvnic
    network: mynetwork
    data_center: datacenter
    network_filter: clean-traffic-gateway
```

Reply:
great thanks